client/apps/edit/components/content2/sections/slideshow/index.coffee

```diff
@@ -0,0 +1,61 @@
+#
+# Slideshow section that appears at the top of a post. Used mainly as a way
+# to drop in in migrated posts.
+#
+
+_ = require 'underscore'
+React = require 'react'
+imagesLoaded = require 'imagesloaded'
+{ getIframeUrl } = require '../../../../../../models/section.coffee'
+{ div, section, ul, li, img, p, strong, iframe } = React.DOM
+
+module.exports = React.createClass
+  displayName: 'SectionSlideshow'
+
+  componentDidMount: ->
+    @props.section.fetchSlideshowArtworks success: => @forceUpdate()
+    # @lockWidths()
```

Comment:
These weren't working so left them commented out -- hoping to deprecate this for real in the near future